hasher-matcher-actioner/webapp/src/Api.tsx

```diff
@@ -324,23 +324,23 @@ export async function fetchHashCount(): Promise<Response> {
 
 // TODO remove the trailing slash from the API URL, then add back the leading slash for /actions/ and /action-rules/ endpoints.
 
+// This class should be kept in sync with python class ActionPerformer
+export type APIActionPerformer = {
```

Comment:
nit: why API prefix?

Reply:
Changed to `Backend` since I think thats more representative